client/src/app/tabs/form/__tests__/FormEditorSpec.js

```diff
@@ -622,7 +622,14 @@ describe('<FormEditor>', function() {
       // then
       const { form } = instance.getCached();
 
-      expect(onActionSpy).to.have.been.calledOnce;
+      var calls = onActionSpy.getCalls();
+
+      calls = calls.filter(function(call) {
+        return call.args[0] === 'lint-tab';
+      });
+
+      // then
+      expect(calls.length).to.be.eql(1);

```

Comment:
```suggestion
      expect(calls).to.have.lengthOf(1);
```

Reply:
So that for failure it says "expected calls to have length of 1" instead of "expected 2 to be equal 1".

Reply:
This is much cleaner. Thank you for the suggestion.